src/main/java/system/Ui.java

```diff
@@ -0,0 +1,205 @@
+                "event ___ /from ___ /to ___\n" +
+                "Please input the date format in: yyyy-mm-dd hhmm.";
+
+        return response;
+    }
+
+    /**
+     * Displays a message saying goodbye as a salutation.
+     */
+    public String goodbye() {
+        return "Bye! Hope to see you again!";
+    }
+
+    /**
+     * Displays a message indicating that the task list is currently empty.
+     */
+    public String emptyList() {
+        return "There are currently zero tasks in the list!";
+    }
+
+    /**
+     * Displays a message indicating that the task does not exist.
+     */
+    public String doesNotExist() {
+        return "Task does not exist!";
+    }
+
+    /**
+     * Displays a message indicating that the task has already been marked.
+     */
+    public String alreadyMarked() {
+        return "Task has already been marked!";
+    }
+
+    /**
+     * Displays a message indicating that the task has already been unmarked.
+     */
+    public String alreadyUnmarked() {
+        return "Task has already been unmarked!";
+    }
+
+    /**
+     * Displays a message indicating that a task has been removed from the task list.
+     * The method outputs details about the removed task, such as the name, status, and relevant date information whenever applicable.
+     * It also displays the updated number of tasks in the list.
+     *
+     * @param t Task that has been removed, which can be an instance of ToDos, Deadlines, or Events.
+     */
+    public String delete_message(Task t) {
+        StringBuilder response = new StringBuilder();
+        DateTimeSystem dateTimeSystem = new DateTimeSystem();
+        response.append("Noted. I've removed this task:\n");
+        if (t instanceof ToDos) {
+            if (t.getCurrentStatus() == Task.Status.MARKED) {
+                response.append("[T][X] ").append(t.getName()).append("\n");
+            } else {
+                response.append("[T][ ] ").append(t.getName()).append("\n");
+            }
+        } else if (t instanceof Deadlines) {
+            if (t.getCurrentStatus() == Task.Status.MARKED) {
+                response.append("[D][X] ").append(t.getName()).append(" (by: ").append(dateTimeSystem.format(t.getDate())).append(")\n");
+            } else {
+                response.append("[D][ ] ").append(t.getName()).append(" (by: ").append(dateTimeSystem.format(t.getDate())).append(")\n");
+            }
+        } else if (t instanceof Events) {
+            if (t.getCurrentStatus() == Task.Status.MARKED) {
+                response.append("[E][X] ").append(t.getName()).append(" (from: ").append(dateTimeSystem.format(t.getStart())).append(" to: ").append(dateTimeSystem.format(t.getEnd())).append(")\n");
+            } else {
+                response.append("[E][ ] ").append(t.getName()).append(" (from: ").append(dateTimeSystem.format(t.getStart())).append(" to: ").append(dateTimeSystem.format(t.getEnd())).append(")\n");
+            }
```

Comment:
not too sure if possible to perhaps refactor these into a seperate function for redability?